Hackney.Shared.Asset/Domain/AssetCharacteristics.cs

```diff
@@ -7,5 +9,17 @@ public class AssetCharacteristics
         public int NumberOfLivingRooms { get; set; }
         public string WindowType { get; set; }
         public string YearConstructed { get; set; }
+        public int NumberOfBedSpaces { get; set; }
+        public int NumberOfCots { get; set; }
+        public int ShowerRoomAmount { get; set; }
+        public int BathroomAmount { get; set; }
+        public string AssetPropertyFolderLink { get; set; }
```

Comment:
What is the folder link? A Google drive folder location?

Reply:
This is a link to a google drive folder